Car.cpp

```diff
@@ -16,3 +20,19 @@ void Car::accelerate(int speed)
         velocity += speed;
     }
 }
+void Car::changeGear(int gear)
+{
```

Comment:
add
```
if(currentGear == gear)
{
 return;
}
```
(a.k.a "early exit")
before try/catch blocks.
It will prevent making unecessary opearations for changing a gear to the same value.